client/src/constants.js

```diff
@@ -0,0 +1,26 @@
+const ImageSequenceType = 'image-sequence';
+const VideoType = 'video';
+
+// Supported File Types
+const videoFileTypes = ['.mp4', '.avi', '.mov', '.mpg'];
+const webFriendlyImageFileTypes = ['.jpg', '.jpeg', '.png'];
+const imageFileTypes = [...webFriendlyImageFileTypes, '.bmp', '.sgi', '.tif', '.tiff', '.pgm'];
+
+// Utility for commonly used regular expressions
+const videoFilesRegEx = new RegExp(`${videoFileTypes.join('$|')}$`, 'i');
+const imageFilesRegEx = new RegExp(`${imageFileTypes.join('$|')}$`, 'i');
+const webFriendlyImageRegEx = new RegExp(`${webFriendlyImageFileTypes.join('$|')}$`, 'i');
```

Comment:
Feel free to disregard for now if I'm looking too far forward.:
Should we consolidate this more into a object like `MediaTypes` which contains the type name, a display name, supported file types.  One source of the supported media types and allows to add stuff like "Satellite Imagery" for stuff like geoTiff and others in the future.

Reply:
Out of scope for this PR, IMO.